### ISSUE TITLE
Problems with Julia v1.0.0

### ISSUE THREAD

**Issue by cormullion** [JuliaAstro/AstroLib.jl#50]
I don't understand this error. Why is Dates a problem here?

```
julia-1.0> using AstroLib
[ Info: Precompiling AstroLib [c7932e45-9af1-51e7-9da9-f004cd3a462b]
ERROR: LoadError: UndefVarError: Dates not defined
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include(::Module, ::String) at ./sysimg.jl:29
 [4] top-level scope at none:2
 [5] eval at ./boot.jl:319 [inlined]
 [6] eval(::Expr) at ./client.jl:389
 [7] top-level scope at ./none:3
in expression starting at /Users/me/.julia/packages/AstroLib/eLfkt/src/AstroLib.jl:7
ERROR: Failed to precompile AstroLib [c7932e45-9af1-51e7-9da9-f004cd3a462b] to /Users/me/.julia/compiled/v1.0/AstroLib/rCD8G.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834

julia-1.0> exit()
Jupiter ~  $ julia1
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.0 (2018-08-08)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia-1.0> using Dates

julia-1.0> using AstroLib
[ Info: Precompiling AstroLib [c7932e45-9af1-51e7-9da9-f004cd3a462b]
ERROR: LoadError: UndefVarError: Dates not defined
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include(::Module, ::String) at ./sysimg.jl:29
 [4] top-level scope at none:2
 [5] eval at ./boot.jl:319 [inlined]
 [6] eval(::Expr) at ./client.jl:389
 [7] top-level scope at ./none:3
in expression starting at /Users/me/.julia/packages/AstroLib/eLfkt/src/AstroLib.jl:7
ERROR: Failed to precompile AstroLib [c7932e45-9af1-51e7-9da9-f004cd3a462b] to /Users/me/.julia/compiled/v1.0/AstroLib/rCD8G.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834
```

**Comment by giordano** [comment]
It seems there have been some late breaking changes before 1.0 (or missing deprecations?), a couple of weeks ago this package passed CI without warnings (coming from the package): https://travis-ci.org/JuliaAstro/AstroLib.jl/jobs/409533509. Also in `Measurements.jl` there are issues not caught by CI a few days ago (https://github.com/JuliaPhysics/Measurements.jl/issues/18).

This week I'm on vacation without a computer, please wait some days :sweat_smile:

**Comment by cormullion** [comment]
enjoy your holiday (and some dark skies perhaps)! 😎 I'm in no hurry. 😂

**Comment by benelsen** [comment]
I suspect you're using the last tagged release? That one does not work in 1.0 (0.7 should be fine), but the master branch does. 
There was one bug in one of the tests (cf #51) but they should now all pass.

**Comment by cormullion** [comment]
i just did `] addAstroLib` so I'm picking up the Registry one. I don't know how often that gets updated, presumably from METADATA every few hours....

**Comment by giordano** [comment]
Well, yes, the last tagged version definitely doesn't support 1.0 :sweat_smile: master branch has support for julia 0.7, but then I went on vacation last week and in the meantime they released julia 1.0 just a few hours after 0.7...

**Comment by giordano** [comment]
I'm going to close this ticket.  The only issue was that last tagged version doesn't support Julia 1.0, but master branch does.  @benelsen fixed an issue in a test with #51
